site/public/css/server.css

```diff
@@ -251,7 +251,8 @@ table tr.table-header {
 }
 
 .table>thead>tr {
```

Comment:
Rather than changing `tr` to `th`, It should be applied to both. Since `tr` and `th` are two different elements. 
I am addressing this in my next commit.

Reply:
Sorry but @xypnox  I want to say change `td` to `tr` as it will directly convert normal  text into bold by default. So no need to add CSS for bolding respective text